src/main/resources/hudson/tasks/_ant/AntTargetNote/DescriptorImpl/outline.jelly

```diff
@@ -21,17 +21,16 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
-
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
-  <l:ajax>
-    <table class='pane' id='console-outline'>
-      <tr>
-        <td class='pane-header'>${%Executed Ant Targets}</td>
-      </tr>
-      <tr>
-        <td id='console-outline-body' />
-      </tr>
-    </table>
-  </l:ajax>
-</j:jelly>
\ No newline at end of file
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+    <l:ajax>
+        <table class='pane' id='console-outline'>
+            <tr>
+                <td class='pane-header'>${%Executed Ant Targets}</td>
+            </tr>
+            <tr>
+                <td id='console-outline-body'/>
+            </tr>
+        </table>
+    </l:ajax>
+</j:jelly>
```

Comment:
/me not convinced about this kind of change: IMO though tempting, we should refrein from reformating unrelated code to make code archeology easier.